support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -10,6 +10,7 @@ export type ChoiceCardsProductSetTestR3Variants = 'control' | 'yellow';
 const contributionsLandingPageMatch = '/(uk|us|eu|au|ca|nz|int)/contribute(/.*)?$';
 const usOnlyLandingPage = '/us/contribute(/.*)?$';
 const ukOnlyLandingPage = '/uk/contribute(/.*)?$';
+export const subsShowcaseAndDigiSubPages = '(/??/subscribe(\\?.*)?$|/??/subscribe/digital(\\?.*)?$)';
```

Comment:
Perhaps we could make `targetPageMatches` take a list of regex strings to split/simplify this slightly. Works this way though.

Reply:
Would be a good idea but maybe not in this PR